client/landing/stepper/declarative-flow/internals/steps-repository/playground/test/index.tsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import PlaygroundStep from '..';
+import { StepProps } from '../../../types';
+import { renderStep, mockStepProps, RenderStepOptions } from '../../test/helpers';
+
+// Be sure to mock this hook to return true so that the step is rendered.
+jest.mock( 'calypso/landing/stepper/hooks/use-is-playground-eligible', () => ( {
+	useIsPlaygroundEligible: () => true,
+} ) );
+
+const renderPlaygroundStep = (
+	props?: Partial< StepProps >,
+	renderOptions?: RenderStepOptions
+) => {
+	const combinedProps = { ...mockStepProps( props ) };
+
+	return renderStep( <PlaygroundStep { ...combinedProps } />, renderOptions );
+};
+
+const launchButton = () => screen.getByRole( 'button', { name: 'Launch on WordPress.com' } );
```

Comment:
I'd rename this to `getLaunchButton` so it conveys more clearly that it retrieves an element.

Alternatively, you could just inline the queries, that would be fine IMO since it keeps the tests readable (some duplication is fine if it makes code more readable and easy to follow).